src/cmd/ksh93/bltins/typeset.c

```diff
@@ -366,10 +359,10 @@ int    b_typeset(int argc,register char *argv[],Shbltin_t *context)
 				flag |= NV_TAGGED;
 				break;
 			case 'u':
-			        if(!isfloat)
-			        {
-				        tdata.wctname = e_toupper;
-				        flag |= NV_LTOU;
+				if(!isfloat)
+				{
+					tdata.wctname = e_toupper;
+					flag |= NV_LTOU;
```

Comment:
I am the author of this code. Please teach me what I did wrong.

Reply:
Same problem with spaces and tabs as show above.